lib/build.ts

```diff
@@ -147,11 +180,6 @@ async function compileOnUnix(nodeVersion: string, targetArch: string) {
 
   const output = path.join(nodePath, 'out/Release/node');
 
-  // https://github.com/mhart/alpine-node/blob/base-7.4.0/Dockerfile#L36
-  if (hostPlatform === 'alpine') {
-    spawnSync('paxctl', ['-cm', output], { stdio: 'inherit' });
-  }
-
```

Comment:
Why we don't need this anymore?

Reply:
https://git.alpinelinux.org/aports/commit/main/nodejs/APKBUILD?id=5f056bf7293a6dcf8871656db7811b6e7e8b134f